main.go

```diff
@@ -119,9 +120,19 @@ func readServerMessages(c *ishell.Shell) {
 	}
 }
 
-func main() {
+func configure(c *ishell.Shell) {
+	historyPath := os.Getenv("PITAYACLI_HISTORY_PATH")
+	if historyPath == "" {
+		historyPath = ".pitayacli_history"
```

Comment:
I guess you want to set historyPath to $HOME/.pitayacli_history, see https://github.com/mitchellh/go-homedir

Reply:
Done!